source/node.go

```diff
@@ -57,7 +57,11 @@ func NewNodeSource(ctx context.Context, kubeClient kubernetes.Interface, annotat
 	nodeInformer.Informer().AddEventHandler(
 		cache.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
-				log.Debug("node added")
+				node, ok := obj.(*v1.Node)
```

Comment:
This is will make an API call to on every node event, which could lead to throttling, even if debug is not enabled.....

should be something like, pseudo code

```go

if loglevel == debug
  do the stuff
```